src/cli.ts

```diff
@@ -30,6 +30,7 @@ program
     })
   )
   .option('-s, --silent', 'silent all logs')
+  .option('--cairo-1', 'Run a Cairo program')
```

Comment:
cairo1 doesn't exist anymore, it's cairo and cairozero/cairo0

why would we need this?

Reply:
Currently the CairoRunner for Cairo Zero and Cairo programs uses different methods (either `fromProgram` or `fromCairo1Program` (should be renamed `fromCairoZeroProgram` and `fromCairoProgram` if any distinction is kept...))

I needed a way to distinguish both programs from the CLI, hence this flag.
Best would be to remove it, otherwise at least rename it `--cairo`. Or the other way around, have a `--cairo-zero` flag which expects a Cairo Zero program and make Cairo programs the default